rhel6/guide/services/dhcp/disabling_dhcp_server/package_dhcp_removed.rule

```diff
@@ -20,4 +20,4 @@ references:
     disa: 366
     nist: CM-7
 
-ocil: '<package-check-macro package="dhcp" />'
+{{{ package_check_complete_ocil_entry(package="dhcp", pkg_system=pkg_system) }}}
```

Comment:
All `ocil` keys need to remain as `ocil: jinja_template`

Reply:
@redhatrises as @matejak pointed out earlier we need the ocil_clause and ocil itself, that can't be achieved with the syntax you suggest. You could workaround it but it would be even more confusing and error prone.

Reply:
Understood. However, `ocil` should be a required key regardless. `ocil_clause` can be an optional key. This brings the content in line with ansible and opencontrol. In addition with the template written as is, there is no way to customize `ocil_clause` which there should be a way to customize it in the template.

Reply:
There are cases when `ocil_clause` is known, so listing the two here explicitly would violate the DRY principle on a massive scale. Also note that the current template macro hierarchy allows you to keep `ocil` and `ocil_clause` defined explicit by using the `ocil: {{{ package_check(package="dhcp", pkg_system=pkg_system) }}}` syntax.